.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-  - "0.10"
+  - '0.10'
   - '0.12'
-  - 'iojs'
+  - 'node'
```

Comment:
Removing `node` would mean that any time there is a new node version, you'll have to go back to the repo to update the .travis.yml.
What about : 
````
   - '0.10'
   - '0.12'
   - '4'
   - '5'
   - 'node'
````
?